tools/st2-inject-trigger-instances.py

```diff
@@ -58,12 +60,27 @@ def _inject_instances(trigger, rate_per_trigger, duration, payload={}):
     while elapsed < duration:
         # print('Dispatching trigger %s at time %s', trigger, date_utils.get_datetime_utc_now())
         dispatcher.dispatch(trigger, payload)
-        delta = random.expovariate(rate_per_trigger)
-        eventlet.sleep(delta)
-        elapsed = (date_utils.get_datetime_utc_now() - start).seconds / 60.0
+
+        if rate_per_trigger:
+            delta = random.expovariate(rate_per_trigger)
+            eventlet.sleep(delta * 0.56)
```

Comment:
What is this 0.56?

Reply:
Yeah, it is a bit of "magic".

For more accurate result, I would need to time ``dispatcher.dispatch`` call and subtract that from delta.

This "magic" number is what I got from my own testing (ran script multiple times, average rate achieved was usually 40-60% smaller than the desired one).

Reply:
I'd remove all this magic numbers and treat the user supplied rate as approximate. Do we really require a precise rate?